code/drasil-data/Data/Drasil/Concepts/Physics.hs

```diff
@@ -161,6 +159,9 @@ isotropy = dccWDS "isotropy" (cn "isotropy") (S "A condition where the" +:+
   phrase value `sOf` S "a" +:+ phrase property +:+ S "is independent of" +:+
   S "the direction in which it is measured.")
 
+chgInVelocity = dccWDS "velocity" (cn "velocity") (S "Change in" +:+ phrase velocity +:+ S "of a"
+   +:+ S "body.")
```

Comment:
it would be better to not split "of a body" this way.

Reply:
fixed